src/util/Injected.js

```diff
@@ -5,15 +5,15 @@ exports.ExposeStore = (moduleRaidStr) => {
     eval('var moduleRaid = ' + moduleRaidStr);
     // eslint-disable-next-line no-undef
     window.mR = moduleRaid();
-    window.Store = Object.assign({}, window.mR.findModule(m => m.default && m.default.Chat)[0].default);
+    window.Store = Object.assign({}, window.mR.findModule(m => m && m.Chat)[1]);
```

Comment:
Just curious - shouldn't it be 0 as in other cases? 
```suggestion
    window.Store = Object.assign({}, window.mR.findModule(m => m && m.Chat)[0]);
```

Reply:
0 only got the function Chat
1 gets the function as the previous default.

![313551828-a6eb6fe6-7895-45ee-b502-ff8fbb7d6e52](https://github.com/pedroslopez/whatsapp-web.js/assets/5648232/e33a7fea-9967-4d98-a872-ce648065e123)

Reply:
TY!

Reply:
i have new [PR](https://github.com/pedroslopez/whatsapp-web.js/pull/2825)
> i think the reason of that is that we have many modules that have Chat function, so we need to check about something else than Chat method to define that this module is the Store, maybe if we check also for AdCollection or Blocklist or any other property we see in the picture, this will do the trick, like this:
> 
> ```
> window.Store = Object.assign({}, window.mR.findModule(m => m && m.Chat && m.Blocklist)[0]);
> ```
> 
> But it needs more test to ensure that Blocklist always found within Store module along the Chat function
> 
> Hope someone can test this and tell us if it does what we need, as currently i am not able to test it from my side !!!